k8s/crd-samples/pool-resource-cluster-1.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: openebs.io/v1alpha1
 kind: CStorPool
 metadata:
-  name: pool1
+  name: pool1-abc
   node: node-host-label
+  labels:
+    "kubernetes.io/hostname": "node-host-label"
```

Comment:
Who understands this label? Is it kubernetes or is it openebs?
Which component understands this label?
I felt, it will be good to explain the motive behind this label in this yaml itself.

Reply:
yes sure

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas